lib/omniauth/strategies/developer_admin.rb

```diff
@@ -0,0 +1,55 @@
+module OmniAuth
+  module Strategies
+    # The DeveloperAdmin strategy is for use in development only.
+    # It allows us to load the admin interface without the need
+    # to talk to an external auth provider.
+    #
+    # You need to set ADMIN_EMAILS to a list with a single email
+    # address in it. When you go to log in it will show a form
+    # with that email address and a default developer name. You
+    # should just be able to click Sign In at that stage.
+    #
+    # It's set up from config/initializers/omniauth but also
+    # requires a POST route in config/routes.rb
+    #
+    #   use OmniAuth::Builder do
+    #     provider :developer_admin
+    #   end
+    class DeveloperAdmin
+      include OmniAuth::Strategy
+
+      DEFAULT_EMAIL = 'delia.veloper@dxw.com'.freeze
+
+      option :name, 'developer_admin'
+      option :email, ENV['ADMIN_EMAILS'] || DEFAULT_EMAIL
```

Comment:
Not sure if it matters, but when I had `ADMIN_EMAILS=` in my .env.development rather than not having it set at all, this didn't work (the default email address didn't get populated), which may confuse devs.

Reply:
It matters. Thanks for spotting, will mitigate.

Reply:
@leeky Have mitigated and nixed a couple of other problems, see commit. Thanks again